### DISCUSSION TITLE
Pikka ..need to have reset on activation

### DISCUSSION THREAD

**Discussion by kjdy9nul** [rsayfutdinov/Pikka#53] [General]
Switched computers, forgot to deactivate. Have not heard back from emails to support@pikka.app.
Please indicate email where I can get some assistance on this...

R

**Comment by claitonllemes** [comment]
I have the same problem, I already sent an email to support and no one responds. I would like a solution to reactivate my app or return to the old version, which in my opinion was much better!

ID: MSM4ZMZK4J

![CleanShot 2024-05-03 at 11 51 07](https://github.com/rsayfutdinov/Pikka/assets/99222756/bb5e350c-3109-4e62-8b26-6e865301b854)
